app/controllers/api/v1/users/good_deeds_controller.rb

```diff
@@ -5,7 +5,8 @@ class Api::V1::Users::GoodDeedsController < ApplicationController
   def create
     new_good_deed = GoodDeed.new(name: params[:deed_name], host_id: params[:user_id], date: params[:date], time: params[:time])
     return unless new_good_deed.save!
+
     new_good_deed.add_participants(params[:attendees], params[:user_id].to_i)
-    render json: GoodDeedsSerializer.new(new_good_deed), status: 201
+    render json: GoodDeedsCreateSerializer.new(new_good_deed), status: 201
```

Comment:
Thanks for explaining why you decided to add a second serializer for Good Deeds. I'm still not sure we need two, but your reasoning made sense.